### ISSUE TITLE
lmfdb/ecnf/templates/ecnf-search-results.html

### ISSUE THREAD

**Issue by edgarcosta** [LMFDB/lmfdb#1173]
Hello,

Looking at the logs, once in a while I see this:
```
Exception on /EllipticCurve/ [GET]
Traceback (most recent call last):
  File "/home/lmfdb/sage3/local/lib/python2.7/site-packages/Flask-0.10.1-py2.7.egg/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/lmfdb/sage3/local/lib/python2.7/site-packages/Flask-0.10.1-py2.7.egg/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/lmfdb/sage3/local/lib/python2.7/site-packages/Flask-0.10.1-py2.7.egg/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/lmfdb/sage3/local/lib/python2.7/site-packages/Flask-0.10.1-py2.7.egg/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/lmfdb/sage3/local/lib/python2.7/site-packages/Flask-0.10.1-py2.7.egg/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/lmfdb/lmfdb-git-prod/lmfdb/ecnf/main.py", line 145, in index
    return elliptic_curve_search(data=request.args)
  File "/home/lmfdb/lmfdb-git-prod/lmfdb/ecnf/main.py", line 371, in elliptic_curve_search
    return search_input_error(info, bread)
  File "/home/lmfdb/lmfdb-git-prod/lmfdb/ecnf/main.py", line 435, in search_input_error
    return render_template("ecnf-search-results.html", info=info, title='Elliptic Curve Search Input Error', bread=bread)
  File "/home/lmfdb/sage3/local/lib/python2.7/site-packages/Flask-0.10.1-py2.7.egg/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/home/lmfdb/sage3/local/lib/python2.7/site-packages/Flask-0.10.1-py2.7.egg/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/home/lmfdb/sage3/local/lib/python2.7/site-packages/Jinja2-2.8-py2.7.egg/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/lmfdb/sage3/local/lib/python2.7/site-packages/Jinja2-2.8-py2.7.egg/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/lmfdb/lmfdb-git-prod/lmfdb/ecnf/templates/ecnf-search-results.html", line 1, in top-level template code
    {% extends 'homepage.html' %}
  File "/home/lmfdb/lmfdb-git-prod/lmfdb/templates/homepage.html", line 4, in top-level template code
    {% import 'color.css' as color %}
  File "/home/lmfdb/lmfdb-git-prod/lmfdb/templates/base.html", line 120, in top-level template code
    {% block body -%}{%- endblock body %}
  File "/home/lmfdb/lmfdb-git-prod/lmfdb/templates/homepage.html", line 191, in block "body"
    {% block content -%}
  File "/home/lmfdb/lmfdb-git-prod/lmfdb/ecnf/templates/ecnf-search-results.html", line 36, in block "content"
    {% if info.query.number %}
  File "/home/lmfdb/sage3/local/lib/python2.7/site-packages/Jinja2-2.8-py2.7.egg/jinja2/environment.py", line 408, in getattr
    return getattr(obj, attribute)
UndefinedError: 'dict object' has no attribute 'query'
```

Best,
Edgar

**Comment by AndrewVSutherland** [comment]
Fixed in pull request #1177.

**Comment by AndrewVSutherland** [comment]
This was a good catch Edgar.  If you or anyone else notices suspicious log entries, be sure to post an issue for it.
